ledger-core/virtual/integration/outgoing_test.go

```diff
@@ -131,30 +126,25 @@ func TestVirtual_CallMethodOutgoing_WithTwicePulseChange(t *testing.T) {
 			CallMethod(objectBGlobal, classB, "Bar", []byte("123")).
 			SetInterference(barIsolation.Interference).
 			SetIsolation(barIsolation.State)
-		objectAExecutionMock := runnerMock.AddExecutionMock("Foo")
-		objectAExecutionMock.AddStart(
-			func(_ execution.Context) {
-				logger.Debug("ExecutionStart [A.Foo]")
-				server.IncrementPulseAndWaitIdle(ctx)
-				secondPulse = server.GetPulse().PulseNumber
-				firstExpectedToken.PulseNumber = secondPulse
-			},
-			&execution.Update{
-				Type:     execution.OutgoingCall,
-				Error:    nil,
-				Outgoing: outgoingCall,
-			},
-		)
-		objectAExecutionMock.AddContinue(
-			func(result []byte) {
-				logger.Debug("ExecutionContinue [A.Foo]")
-				require.Equal(t, []byte("finish B.Bar"), result)
-			},
-			&execution.Update{
-				Type:   execution.Done,
-				Result: requestresult.New([]byte("finish A.Foo"), objectAGlobal),
-			},
-		)
+
+		runnerMock.AddExecutionMock("Foo").AddStart(func(_ execution.Context) {
+			t.Log("ExecutionStart [A.Foo]")
```

Comment:
https://insolar.atlassian.net/wiki/spaces/DEV/pages/1431568423/Core+v2+-+Logger+in+tests

Reply:
fixed